samples/kvsWebRTCClientViewer.c

```diff
@@ -173,6 +181,9 @@ INT32 main(INT32 argc, CHAR *argv[])
         MUTEX_UNLOCK(pSampleConfiguration->sampleConfigurationObjLock);
     }
 
+    if(pSampleConfiguration->enableFileLogging) {
```

Comment:
Oh, one more thing... Make sure that if the createFileLogger fails, we don't attempt to free it. I believe the free is idempotent so it will evaluate to No-op if the file logger is NULL but double check

Reply:
Fixed.